testing/dart/fragment_shader_test.dart

```diff
@@ -20,6 +21,26 @@ void main() async {
     return true;
   }());
 
+  test('impellerc produces reasonable JSON encoded IPLR files', () async {
+    final String path = Platform.environment['FLUTTER_FRAGMENT_SHADER_TEST_PATH']!;
+    final Object? rawData = convert.json.decode(
+      File('$path/gen/flutter/lib/ui/fixtures/shaders/iplr-json/ink_sparkle.frag.iplr').readAsStringSync());
```

Comment:
This logic essentially already exists in `shaderDirectory()` in `shader_test_file_utils.dart` in this directory. No need to add a new env var.